code/modules/mob/living/silicon/robot/robot_damage.dm

```diff
@@ -4,8 +4,15 @@
 		stat = CONSCIOUS
 		return
 	health = maxHealth - (getBruteLoss() + getFireLoss())

```

Comment:
Whoa, whoa, what? 

Somehow missed this. But uh.
Robot/Life() specifically recalculated it the other way around... why bay why